aptos-move/aptos-vm/src/transaction_validation.rs

```diff
@@ -57,6 +57,11 @@ pub static APTOS_TRANSACTION_VALIDATION: Lazy<TransactionValidation> =
         unified_prologue_name: Identifier::new("unified_prologue").unwrap(),
         unified_prologue_fee_payer_name: Identifier::new("unified_prologue_fee_payer").unwrap(),
         unified_epilogue_name: Identifier::new("unified_epilogue").unwrap(),
+
+        unified_prologue_v2_name: Identifier::new("unified_prologue_v2").unwrap(),
+        unified_prologue_fee_payer_v2_name: Identifier::new("unified_prologue_fee_payer_v2")
+            .unwrap(),
+        unified_epilogue_v2_name: Identifier::new("unified_epilogue_v2").unwrap(),
```

Comment:
what's the schedule of daa? if this one lands before enabling daa we can just modify unified_xxlogue..
cc @igor-aptos

Reply:
In one of the comment, Igor mentioned dAA will be enabled in v1.29. Not sure if that changed later on.